src/west/commands/project.py

```diff
@@ -74,6 +74,7 @@ def do_run(self, args, user_args):
             shutil.move(args.cache, project.abspath)
 
             _update_key(config, 'manifest', 'path', project.path)
+        _update_key(config, 'zephyr', 'zephyr-base', project.path)
```

Comment:
I think this should probably be:

```
[zephyr]
base=/path/to/zephyr
```

Reply:
fine with me.
Will wait a little before changing, to give other a chance to comment also ;)

Reply:
changed to: `base`
This also makes it much nicer where #215 goes in, as you will then write:
`west config zephyr.base` to fetch the value ;)